Example/metro.config.js

```diff
@@ -50,12 +50,6 @@ const config = {
     // to various errors. To mitigate this we define below custom request resolver, hijacking requests to conflicting modules and manually
     // resolving appropriate files. **Most likely** this can be achieved by proper usage of blockList but I found this method working ¯\_(ツ)_/¯
     resolveRequest: (context, moduleName, platform) => {
-      if (moduleName.startsWith('@react-navigation')) {
-        // For some reason, react-navigation packages don't want to resolve from
-        // the project's node_modules, so we need to use standard Metro resolver.
-        return context.resolveRequest(context, moduleName, platform);
-      }
-
```

Comment:
Why this has been removed? Were there any errors regarding resolving this module? I'm just curious 🤔

Reply:
After deleting the package.json file from `apps/examples` this code had to be deleted as the app couldn't find the react navigation packages